src/types/logs.ts

```diff
@@ -1,5 +1,6 @@
-type ProcessedLogLine = number | number[];
+type ExpandedLines = Array<[number, number]>;
```

Comment:
nit. So we can use ExpandedLine in the future if we need it.
```suggestion
type ExpandedLine = [number, number];
type ExpandedLines = ExpandedLine[];
```